### PR TITLE
rhtpa

### PR DESCRIPTION
**Pull request by desmax74**
RHTPA Trusted-service

@ralphbean

### CODE REVIEW

**Comment by desmax74** [comment]
@ralphbean  how can we add here more than one releasePlan https://github.com/redhat-appstudio/tenants-config/tree/main/cluster/stone-prd-rh01/tenants/trusted-content-tenant if another release plan is present ?

File: cluster/stone-prd-rh01/tenants/rhtpa-tenant/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - release-plan.yaml
+namespace: trusted-content-tenant
```
Comment (ralphbean) [inline cluster/stone-prd-rh01/tenants/rhtpa-tenant/kustomization.yaml:5]:
```suggestion
namespace: rhtpa-tenant
```

This needs to be updated and then it needs a rebuild of the manifests script to pass CI.

**Comment by desmax74** [comment]
@ralphbean our pipelines are executed under **trusted-content-tenant**

**Comment by ralphbean** [comment]
That looks better. Thanks!

**Pull request merged**